.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,17 @@
+name: gitleaks
```

Comment:
let's rename the file to scan-secrets.yml and change the name to Scan for secrets